app/components/VerticalSwipeView/VerticalSwipeView.data.ts

```diff
@@ -0,0 +1,3 @@
+import { WINDOW_HEIGHT } from '../../constants/window';
+
+export const SCROLL_VIEW_HEIGHT = (WINDOW_HEIGHT - 78) * 2;
```

Comment:
What is the 78 magic number? :D 
Did you test it with Android?
Please, make this number as constant here (without the export maybe)